hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/execution/SparkLazyInsertIterable.java

```diff
@@ -67,7 +67,7 @@ protected List<WriteStatus> computeNext() {
     // Executor service used for launching writer thread.
     HoodieExecutor<List<WriteStatus>> bufferedIteratorExecutor = null;
     try {
-      Schema schema = new Schema.Parser().parse(hoodieConfig.getSchema());
+      Schema schema = new Schema.Parser().parse(hoodieConfig.getWriteSchema());
```

Comment:
can we add java docs to call out why we use writeSchema instead of just getSchema